addons/project/models/project_task.py

```diff
@@ -993,7 +993,7 @@ def _ensure_fields_are_accessible(self, fields, operation='read', check_group_us
         if fields and (not check_group_user or self.env.user._is_portal()) and not self.env.su:
             unauthorized_fields = set(fields) - (self.SELF_READABLE_FIELDS if operation == 'read' else self.SELF_WRITABLE_FIELDS)
             if unauthorized_fields:
-                unauthorized_field_list = format_list(self.env, list(unauthorized_fields))
+                unauthorized_field_list = list(unauthorized_fields)
```

Comment:
`list` wrapping looks unnecessary now.

Note unrelated to PR: it would be best to update the `set` to `OrderedSet` if we want a deterministic order.

Reply:
Removed the `list`. I'm not changing the other thing, since I guess it doesn't matter much. And out of translations scope.